### ISSUE TITLE
Pre-trained Generator Weights

### ISSUE THREAD

**Issue by fahadshamshad** [abhishekaich27/GAMA-pytorch#3]
Hey, thanks for the great work. Could you upload the pre-trained generator weights? After training, we found that the results of GAMA are better than the baselines. However, they are lower than the numbers reported in the paper. Specifically, in Table 2, using VGG-19 as the surrogate and DenseNet121 as the black-box, we observed 53.65, whereas the paper reported 44.58. We used the parameters as outlined in the paper. Could you possibly assist us in determining where we might have gone wrong?

**Comment by abhishekaich27** [comment]
Hey @fahadshamshad , I do not have access to the original pre-trained weights anymore. Can you check if the result holds after multiple runs? Also, may be training it longer. 
As an observation through my experiments, VGG19 should result in strong perturbations.

**Comment by hoonsyang** [comment]
Hi @abhishekaich27, I'm also having the same issue. My run shows better results than baselines but worse than the reported values in the paper. Are they the averaged values from multiple runs? The code seems to not able to control the random seed for reproducibility.

**Comment by abhishekaich27** [comment]
@hoonsyang Can you report your numbers? My results are after 1 run.
